### ISSUE TITLE
bug list

### ISSUE THREAD

**Issue by TheBricktop** [tentone/nunuStudio#401]
# Few things that breaks 

##### Description

- added script that was directly ported from portal example to make somewhat physical fps controller
-it was broken obviously but when the error happened there was no way to determine which line was broken, and even after deleting the script the run button was stuck at stop, and shown the last error message. had to recreate the scene. This would be a minor issue but having to recreate everthing from scratch prevents me from thinking about nunustudio for making something bigger than a simple demo.
- when adding postfx to camera the antialiasing from the base camera gets overridden and disabled, so even with bloomfx lines look severely jagged
- dropping js file on the object does nothing, i have to create new script object and then copy and paste the js file content to it so its pretty useless to import js in most of the times.
- it kida sad casuse it will warp me back to the unity or aframe.
##### Version
 - <VERSION>0.97.8 <TIMESTAMP>

##### Platform
 - Windows [X]

**Comment by tentone** [comment]
Hi Thanks a lot for reporting these.

I have been working on more detailed info on the console, and i been workin on solution to link console errors to the script editor for provide detailed code location of the error.

The PostFX in webgl pretty much forces the antialiasing disabled on WebGL 1.0 we cannot render to a buffer with antialiasing on. In postfx you have to use one of the postfx aa solutions.

To be honest with you, if you're doing some significative commercial work, i would prefer to go with one of the libs directly (threejs, babylon, etc) or possibly with one of major engines, unity seems to be the one with best web support. Godot also works pretty well but its more limited in the web.

Currently im working on nunuStudio less than 2H a week, development its going really slow and i cannot provide solutions fast enough for most stuff.

Thanks a lot for the support and for undestanding.

**Comment by TheBricktop** [comment]
Yeah, didnt mean to sound harsh, and I think Your editor is still one of the better suited for the commercial work, and I think it should be merged somehow to the main three.js editor cause it allows the designers to create better looking scenes.
